Cargo.lock

```diff
@@ -36,9 +36,9 @@ checksum = "cf1de2fe8c75bc145a2f577add951f8134889b4795d47466a54a5c846d691693"
 
 [[package]]
```

Comment:
I've noticed that we don't update the `vmm` Cargo.lock. Should we actually remove it?

Reply:
Yes. That lock is actually ignored, and I think it was added by mistake.